src/LibraryManager.Vsix/UI/Models/InstallDialogViewModel.cs

```diff
@@ -259,16 +259,23 @@ public ILibrary SelectedPackage
 
         private void RefreshFileSelections()
         {
-            SelectedProvider.GetCatalog().GetLibraryAsync(_packageId, CancellationToken.None).ContinueWith(x =>
+            try
             {
-                if (x.IsFaulted || x.IsCanceled)
+                SelectedProvider.GetCatalog().GetLibraryAsync(_packageId, CancellationToken.None).ContinueWith(x =>
                 {
-                    SelectedPackage = null;
-                    return;
-                }
+                    if (x.IsFaulted || x.IsCanceled)
+                    {
+                        SelectedPackage = null;
+                        return;
+                    }
 
-                SelectedPackage = x.Result;
-            });
+                    SelectedPackage = x.Result;
+                });
+            }
+            catch
+            {
+                // TO DO: What needs to be done here
```

Comment:
@bhsubra, added a ry/catch here since GetLibraryAsync can throw. Please review and see what needs to be done here.